ios/RNSScreenStack.mm

```diff
@@ -120,16 +177,7 @@ - (void)navigationController:(UINavigationController *)navigationController
     }
   }
   [RNSScreenStackHeaderConfig willShowViewController:viewController animated:animated withConfig:config];
-}
-
-- (void)navigationController:(UINavigationController *)navigationController
```

Comment:
it should also not be paper specific. I think it will be easiest to just add this `onFinishTransitioning` to ScreenStack on Fabric and remove all the code `if`ing it.

Reply:
Noted down. Leaving it for new PR.